package.json

```diff
@@ -17,7 +19,9 @@
     "grunt-contrib-watch": ">=0.6.x",
     "grunt-nunjucks": ">=0.1.x",
     "grunt-processhtml": ">=0.3.x",
-    "jshint": "^2.6.0"
+    "jshint": "^2.6.0",
+    "lodash": "^3.7.0",
+    "marionette-client": "^1.7.3"
```

Comment:
You're right, `chai` is not needed as the only assertion is `assert.equal` which is included in Node's standard lib. Regarding `es6-promise`, the tests are not run in the browser but in Node.